### ISSUE TITLE
Can't create a server with ServerInstances

### ISSUE THREAD

**Issue by Vulqa** [Skungee/Skungee#26]
Hello,

When i want to create a server with ServerInstances i got this error, my ServerInstances nodes in the Skungee configurations are enable.

```
06:40:30 [GRAVE] java.lang.NoClassDefFoundError: me/limeglass/skungee/objects/ServerInstancesPacket
06:40:30 [GRAVE]     at me.limeglass.serverinstances.a.b.run(Unknown Source)
06:40:30 [GRAVE]     at java.lang.Thread.run(Unknown Source)
06:40:30 [GRAVE] Caused by: java.lang.ClassNotFoundException: me.limeglass.skungee.objects.ServerInstancesPacket
06:40:30 [GRAVE]     at net.md_5.bungee.api.plugin.PluginClassloader.loadClass0(PluginClassloader.java:53)
06:40:30 [GRAVE]     at net.md_5.bungee.api.plugin.PluginClassloader.loadClass(PluginClassloader.java:27)
06:40:30 [GRAVE]     at java.lang.ClassLoader.loadClass(Unknown Source)
06:40:30 [GRAVE]     ... 2 more
```

My spigot's version: 1.12.2
My skript's version: 2.3-beta2
My skungee's version: 1.0.7d
My bungeecord's version: http://prntscr.com/lfbdcd

**Comment by TheLimeGlass** [comment]
I need your SkUnity forum account name and ID before I can further help you out.

Make sure you have fully downloaded the latest of both addons. Also 2.3 has not been tested yet.

**Comment by TheLimeGlass** [comment]
You have any consolation errors? The problem is that ServerInstances wasn't found.

**Comment by Vulqa** [comment]
@TheLimeGlass 
No error message except the one who appears when i want to create a server

**Comment by TheLimeGlass** [comment]
Still persists?

**Comment by TestingUser661** [comment]
Same error.

**Comment by TheLimeGlass** [comment]
Fixed in 1.0.1 of ServerInstances. Live on Github, release will be soon on SkUnity.
